maptivist/src/main/webapp/script.js

```diff
@@ -191,8 +191,14 @@ function drawMarker({id, latitude, longitude, title, description, links, categor
 /**
 * Adds an infowindow based on the marker creation form
  */
+
+function createInfowindow(position, marker) {
+  //Set info window content from form
+  var title = document.getElementById('marker-title').value;
+
```

Comment:
Looks like a bad merge?

Double check that the info windows still pop up?

Reply:
I think so, I fixed it and the page still opens.